Sources/SwiftInspectorVisitors/Tests/NestableTypeVisitorSpec.swift

```diff
@@ -953,6 +954,12 @@ final class NestableTypeVisitorSpec: QuickSpec {
           expect(matching.count) == 1
         }
 
+        it("finds the property declared in FooEnum.FooFooClass") {
+          let fooFooClass = self.sut.classes.filter { $0.name == "FooFooClass" }
+          let property = fooFooClass.first?.properties.first
+          expect(property?.name) == "someFooFooClassProperty"
```

Comment:
let's grab the type here too, and maybe even add a test that we don't find the type if there is no type? (that test would be testing more that we don't break entirely rather than enforcing that we don't want to infer types. maybe add a TODO to that test to infer types?)

Reply:
Similar to https://github.com/fdiaz/SwiftInspector/pull/100#discussion_r667136387